### PR TITLE
Only offer selinux labels on systems that support selinux

### PR DESCRIPTION
**Pull request by marusak**

### CODE REVIEW

File: src/ImageRunModal.jsx [hunk 1]
```diff
@@ -106,7 +106,7 @@ const EnvVar = ({ id, item, onChange, idx, removeitem, additem }) =>
         </>
     );
 
-const Volume = ({ id, item, onChange, idx, removeitem, additem }) =>
+const Volume = ({ id, item, onChange, idx, removeitem, additem, options }) =>
```
Comment (martinpitt) [inline src/ImageRunModal.jsx:109]: I'd rather leave out the `options` indirection and specify them as direct properties. More robust and easier to check with propTypes and such if they are flat.

That's not a veto though, if you prefer that it's fine.

File: src/app.jsx [hunk 2]
```diff
@@ -406,6 +407,9 @@ class Application extends React.Component {
                     }
                 })
                 .fail(e => console.log("Could not read $XDG_RUNTIME_DIR: ", e.message));
+        cockpit.spawn("selinuxenabled")
```
Comment (martinpitt) [inline src/app.jsx:410]: I checked that `selinuxenabled` works as non-root. But I think this should use `{ error: "ignore" }` to not spam the journal with "no such command" etc. errors.

File: src/ImageRunModal.jsx (hunk 1 above)
Comment (marusak) [reply src/ImageRunModal.jsx:109]: The thing is that this is wrapped with `DynamicListForm` which is used for different components. Adding random options for whichever component we would need them would make it rather messy, thus I decided  to go with general `options` so any component can specify what they need without making this wrapper one big mess.

File: src/app.jsx (hunk 2 above)
Comment (marusak) [reply src/app.jsx]: updated

File: src/ImageRunModal.jsx (hunk 1 above)
Comment (martinpitt) [reply src/ImageRunModal.jsx:109]: Makes sense, thanks!

**Comment by marusak** [comment]
Fedora is green, debian got beyond selinux things. It is broken on exec session closing immediately, but did not figure that one out yet - but it is surely unrelated to this PR.

**Pull request merged**